HerculesStudio-manifest.yaml

```diff
@@ -0,0 +1,21 @@
+---
+app-id: com.jacobdekel.HerculesStudio
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+command: HerculesStudio
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
+- "--socket=wayland"
+- "--filesystem=host"
```

Comment:
as I said previously, this is a very broad filesystem permission.

Reply:
Right. Sorry again 🙁